test/integration/components.cc

```diff
@@ -1192,6 +1193,98 @@ TEST_F(ComponentsTest, ParentLinkName)
   EXPECT_EQ("comp3", comp3.Data());
 }
 
+//////////////////////////////////////////////////
+TEST_F(ComponentsTest, ParticleEmitter)
```

Comment:
This test doesn't seem related to this PR. Bad merge?

Reply:
Related to my [other comment](https://github.com/ignitionrobotics/ign-gazebo/pull/736#discussion_r609837869), I was re-ordering a few tests to follow alphabetical order.